package.json

```diff
@@ -83,6 +82,7 @@
     "import-fresh": "^3.0.0",
     "imurmurhash": "^0.1.4",
     "is-glob": "^4.0.0",
+    "js-sdsl": "^4.1.4-beta.0",
```

Comment:
is it going to be a stable version?

Reply:
Yes, the next week we will releaesd the official version if there no bug reports.